app/models/foreman_google/gce.rb

```diff
@@ -117,6 +117,17 @@ def available_images(filter: nil)
       client.images(filter: filter)
     end
 
+    def vms(attrs = {})
+      vms = client.instances(zone, attrs).map do |vm|
+        ForemanGoogle::GoogleCompute.new client: client,
+          zone: zone,
+          identity: vm.id,
+          instance: vm
+      end
+
+      GoogleCloudCompute::VmsWrapper.new(vms)
```

Comment:
I'd leve the VM wrapping on the collection? So it has something to do.

```suggestion
      GoogleCloudCompute::<CollectionName>.new(client.instances(zone, attrs))
```